src/components/RestaurantDetails/RestaurantDetails.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable jsx-a11y/img-redundant-alt */
+//Libraries
+import React from 'react';
+
+//Helpers
+import fetchData from '../../api/fetchData';
```

Comment:
don't import it as fetchData it's very generic.....
you can name it : **fetchRestaurantDetails**